target/linux/realtek/dts-5.10/rtl8382_panasonic_m16eg-pn28160k.dts

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "rtl838x.dtsi"
+#include "rtl83xx_panasonic_mxxeg-pn28xx0k.dtsi"
+
+#include <dt-bindings/interrupt-controller/irq.h>
+
+/ {
+	compatible = "panasonic,m16eg-pn28160k", "realtek,rtl8382-soc";
+	model = "Panasonic Switch-M16eG PN28160K";
+
+	aliases {
+		led-boot = &led_status_eco_green;
+		led-failsafe = &led_status_eco_amber;
+		led-running = &led_status_eco_green;
+		led-upgrade = &led_status_eco_green;
+	};
+
+	/*
+	 * sfp0/1 are "combo" port with each TP port (23/24), and they are
+	 * connected to the RTL8218FB. Currently, there is no support for
+	 * the chip and only TP ports work by the RTL8218D support.
+	 */
+	sfp0: sfp-p23 {
+		compatible = "sff,sfp";
+		i2c-bus = <&i2c0>;
+		tx-fault-gpio = <&gpio1 0 GPIO_ACTIVE_HIGH>;
+		tx-disable-gpio = <&gpio1 1 GPIO_ACTIVE_HIGH>;
+		mod-def0-gpio = <&gpio1 2 GPIO_ACTIVE_LOW>;
+		los-gpio = <&gpio1 3 GPIO_ACTIVE_HIGH>;
+	};
+
+	sfp1: sfp-p24 {
+		compatible = "sff,sfp";
+		i2c-bus = <&i2c1>;
+		tx-fault-gpio = <&gpio1 4 GPIO_ACTIVE_HIGH>;
+		tx-disable-gpio = <&gpio1 5 GPIO_ACTIVE_HIGH>;
+		mod-def0-gpio = <&gpio1 6 GPIO_ACTIVE_LOW>;
+		los-gpio = <&gpio1 7 GPIO_ACTIVE_HIGH>;
+	};
+};
+
+&leds {
+	led_status_eco_amber: led-5 {
+		label = "amber:status_eco";
+		gpios = <&gpio2 1 GPIO_ACTIVE_LOW>;
+		color = <LED_COLOR_ID_AMBER>;
+		function = LED_FUNCTION_STATUS;
+		function-enumerator = <1>;
+	};
+
+	led_status_eco_green: led-6 {
+		label = "green:status_eco";
+		gpios = <&gpio2 2 GPIO_ACTIVE_LOW>;
+		color = <LED_COLOR_ID_GREEN>;
+		function = LED_FUNCTION_STATUS;
+		function-enumerator = <2>;
+	};
+};
+
+&i2c_gpio_0 {
+	scl-gpios = <&gpio0 0 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+	sda-gpios = <&gpio0 1 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+};
+
+&i2c_gpio_1 {
+	scl-gpios = <&gpio0 12 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+	sda-gpios = <&gpio0 13 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+};
+
+&gpio2 {
+	interrupt-controller;
+	#interrupt-cells = <2>;
+	interrupt-parent = <&gpio0>;
+	interrupts = <2 IRQ_TYPE_EDGE_FALLING>;
+
+	/* GPIO12 (IO1_4): RTL8218FB */
+	ext_switch_reset {
+		gpio-hog;
+		gpios = <12 GPIO_ACTIVE_HIGH>;
+		output-high;
+		line-name = "ext-switch-reset";
+	};
+};
+
+&i2c_switch {
+	i2c0: i2c@0 {
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0>;
+	};
+
+	i2c1: i2c@1 {
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <1>;
+	};
+};
+
+&ethernet0 {
+	mdio-bus {
+		compatible = "realtek,rtl838x-mdio";
+		regmap = <&ethernet0>;
```

Comment:
I'm not sure if I noticed this before, and now I can't find any reference to "regmap" in `target/linux/realtek/files-5.10/`, nor in the upstream mdio binding. Do you know how this property is used?

Reply:
After checking, it seems that it existed from at least "[2b88563ee5 realtek: update the tree to the latest refactored version](https://github.com/openwrt/openwrt/commit/2b88563ee5aafd9571d965b7f2093a0f58d98a31)". I was taking it over by inertia, but it certainly looks like it has nothing to do and is unnecessary.

Reply:
OK, then you should be able to leave it out here, and it can be removed from the other DT files to prevent more copy-pasting. To be clear, I'm not asking you to do that, you can just focus on these patches.